src/index.ts

```diff
@@ -16,6 +16,11 @@ const createMigrationTool = (): number => {
         .example('kontent', 'migration run --name migration01 --project-id <YOUR_PROJECT_ID> --api-key <YOUR_MANAGEMENT_API_KEY>')
         .example('kontent', 'migration run --name migration01 --environment DEV --debug true')
         .example('kontent', 'migration run --all --environment DEV')
+
+        .example('kontent', 'backup --action backup --project-id <YOUR_PROJECT_ID> --api-key <YOUR_MANAGEMENT_API_KEY>')
```

Comment:
Running this script creates zip with filename "undefined.zip". Can we setup some default friendly zip name (using e.g. current date & time?)